index.js

```diff
@@ -1,3 +1,3 @@
 // Run all scripts with ESM syntax support.
-
+require('dotenv').config();
 module.exports = require('esm')(module /* , options */)('./server.js'); // Start server
```

Comment:
based on the current environment we need to read environment variables from the corresponding .env file. 

the path of this .env file needs to be provided to config as a parameter.

```
require('dotenv').config({ path: dotEnvFile });
```